code/lib/create-storybook/src/initiate.ts

```diff
@@ -439,13 +450,15 @@ export async function doInitiate(options: CommandOptions): Promise<
     selectedFeatures.add('docs');
     if (isInteractive) {
       selectedFeatures.add('test');
+      selectedFeatures.add('onboarding');
```

Comment:
is this feature shown in the user prompts when running sb init?

Reply:
will it respect the global config when users decide not to use onboarding anymore in their machines?

Reply:
Yeah that makes sense to me. We might want to change it in the future if we provide some kind of onboarding for new users who did not set up the project. But we can cross that bridge when we come to it.